frontend/src/components/eMIB/Emib.jsx

```diff
@@ -53,12 +78,84 @@ class Emib extends Component {
     }
   };
 
-  openPopup = () => {
-    this.setState({ showPopupBox: true });
+  openSubmitPopup = () => {
+    this.setState({ showSubmitPopup: true });
   };
 
   closePopup = () => {
-    this.setState({ showPopupBox: false });
+    this.setState({ showSubmitPopup: false });
```

Comment:
I think there's actually no need to have multiple setState's in a row. You should be able to do this here and below:
```
this.setState({
  showSubmitPopup: false,
  showQuitPopup: fase
});
```

Reply:
Simplified all the multiple **setState**.
See commit #https://github.com/code-for-canada/project-thundercat/pull/129/commits/350483793ba562d8e94259fb53298e69e55e60bf